addons/binding/org.openhab.binding.astro/README.md

```diff
@@ -101,14 +71,44 @@ Available events:
 
 Offsets:
 
-*For each event group you can optionally configure an offset in the channel properties for the corresponding thing.*
+For each event group you can optionally configure an offset in minutes. The offset must be configured in the channel properties for the corresponding thing. The minimum allowed offset is -1440 and the maximum allowed offset is 1440.
 
+## Full Example
+
+Things:
+
+```
+astro:sun:home  [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60 ]
+astro:moon:home [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60 ]
+```
+or optionally with an offset
 ```
 astro:sun:home [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60 ] {
     Channels:
         Type rangeEvent : rise#event [
             offset=-30
         ]
 }
+astro:moon:home [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60 ]
 ```
 
+Items:
+
+```
+DateTime Sunrise_Time  "Sunrise [%1$tH:%1$tM]"  { channel="astro:sun:home:rise#start" }
+DateTime Sunset_Time   "Sunset [%1$tH:%1$tM]"   { channel="astro:sun:home:set#start" }
+Number   Azimuth       "Azimuth"                { channel="astro:sun:home:position#azimuth" }
+Number   Elevation     "Elevation"              { channel="astro:sun:home:position#elevation" }
+String   MoonPhase     "MoonPhase"              { channel="astro:moon:home:phase#name" }
```

Comment:
While you are at it, could your "correct" these items?
`"Azimuth [%.1f °]"`
`"Elevation [%.1f °]"`
`"Moon Phase [%s]"`